solutions/security-enforced/DA-types.md

```diff
@@ -0,0 +1,97 @@
+# Configuring complex inputs in Event Notifications
```

Comment:
No need to duplicate this, its already in fully-configurable -we will just point variable description to that one